examples/js/vr/HelioWebXRPolyfill.js

```diff
@@ -4,189 +4,189 @@
 
 if ( /(Helio)/g.test( navigator.userAgent ) && "xr" in navigator ) {
 
-	if ( 'isSessionSupported' in navigator.xr ) return;
```

Comment:
Can you please ensure to only use hard tabs in this file? It seems hard and soft tabs are mixed now which ruins proper formatting.

Reply:
Yes please. Check the `files changed` section to make sure your pr is clean:
https://github.com/mrdoob/three.js/pull/17838/files

Reply:
Updated.